library/src/main/java/com/github/pwittchen/prefser/library/Prefser.java

```diff
@@ -220,7 +236,10 @@ public T call(String s) {
      *
      * @param sharedPreferences instance of SharedPreferences to be observed
      * @return Observable with String containing key of the value in SharedPreferences
+     *
+     * @deprecated Create Prefser instance per SharedPreferences and use {@link #observeChangedKey()}.
```

Comment:
good point in JavaDoc. Maybe `observeChangedKey()` could be a better name for this method? We can discuss that.